### PR TITLE
Update to Bot API 5.0

### PR DESCRIPTION
**Pull request by KnorpelSenf**
Title says it all.

API release notes are to be found under https://core.telegram.org/bots/api#november-4-2020

### CODE REVIEW

File: methods.d.ts [hunk 1]
```diff
@@ -78,22 +82,35 @@ export interface Telegram {
...
     Please note that this parameter doesn't affect updates created before the call to the setWebhook, so unwanted updates may be received for a short period of time. */
     allowed_updates?: readonly String[];
+    /** Pass True to drop all pending updates */
+    drop_pending_updates?: Boolean;
   }): True;
 
-  /** Use this method to remove webhook integration if you decide to switch back to getUpdates. Returns True on success. Requires no parameters. */
-  deleteWebhook(): True;
+  /** Use this method to remove webhook integration if you decide to switch back to getUpdates. Returns True on success. */
+  deleteWebhook(args: {
```
Comment (wojpawlik) [inline methods.d.ts]: To avoid breaking:

```suggestion
  deleteWebhook(args?: {
```

File: methods.d.ts [hunk 2]
```diff
@@ -317,16 +392,20 @@ export interface Telegram {
...
+      InputMediaAudio | InputMediaDocument | InputMediaPhoto | InputMediaVideo
+    >;
     /** Sends the messages silently. Users will receive a notification with no sound. */
     disable_notification?: Boolean;
-    /** If the messages are a reply, ID of the original message */
+    /** If messages are a reply, ID of the original message */
     reply_to_message_id?: Integer;
+    /** Pass True, if the message should be sent even if the specified replied-to message is not found */
+    allow_sending_without_reply?: Boolean;
   }): Array<Message.PhotoMessage | Message.VideoMessage>;
```
Comment (wojpawlik) [inline methods.d.ts]:
```suggestion
  }): Array<Message.PhotoMessage | Message.VideoMessage | Message.DocumentMessage | Message.AudioMessage>;
```

Or just

```suggestion
  }): Array<Message>;
```

**Review by wojpawlik — CHANGES_REQUESTED** [review]
`sendVoice` is missing `caption_entities`.

**Comment by KnorpelSenf** [comment]
@wojpawlik thank you for the thorough look! Those issues were well-spotted. Merging.

**Pull request merged**